ideas/310-core-team-proposal.md

```diff
@@ -72,103 +73,123 @@ See https://github.com/status-im/status-react/issues/6757 as an example of a sim
 
 The projects list should have links to umbrella issues for each projects.
 
-### Meetings & Philosophy
+## Research
 
-Process can be revisited through retrospective. Probably, the process should go to a github repository as a markdown file and using PRs to update it.
+### 0. Meta (what this team will be doing?)
 
-Taking as little of the parallel projects as possible. For example, we have to do maintenance anyway, so adding one other project as a focus area should be okay.
+Completion by December, 19th (1 week)
 
-Weekly sync-up calls
-- goals for the week
-- focus area, progress
-- post-meeting report in a Discuss thread
+Goals:
+- Figure out the list of tasks and top UX issues that needs to be fixed
+- Rank them by priority (what should we do first)
 
-Post-project retrospective with post-mortem
-- when the project milestone is done/failed
-- when we are deciding to switch to another focus area
+### 3. Improve Security using native views
 
-[DRIs](https://www.forbes.com/sites/quora/2012/10/02/how-well-does-apples-directly-responsible-individual-dri-model-work-in-practice/#38840682194c) for focus areas / maintenance tasks:
-- single point of contact
-- focused on one thing (one can be a DRI only to one focus area/maintenance task)
+Completion by ???
```

Comment:
@PombeirP I need your help with estimating native views research phase. What do you think?